src/runtime_intrinsics.c

```diff
@@ -11,6 +11,7 @@
 #include "julia.h"
 #include "julia_internal.h"
 #include "APInt-C.h"
+#include "stdio.h"
```

Comment:
necessary change?

Reply:
Oops, that was there from debugging. Fixed.